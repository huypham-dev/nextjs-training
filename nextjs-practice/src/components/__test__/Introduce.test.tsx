import { shallow } from "enzyme";
import Intro from "../Introduce";

describe("Intro", () => {
  it("Snapshot", () => {
    const component = shallow(<Intro />);

    expect(component).toMatchSnapshot();
  });
});
