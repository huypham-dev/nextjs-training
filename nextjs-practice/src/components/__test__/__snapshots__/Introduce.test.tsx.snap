// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro Snapshot 1`] = `
<div
  className="mx-auto max-w-6xl bg-white shadow-xl py-10 mb-10"
>
  <div
    className="max-w-5xl mx-auto p-8 flex justify-start items-center"
  >
    <div
      className="flex-1"
    >
      <Memo(Typography)
        className="font-display-italic font-extrabold text-5xl md:text-6xl italic leading-none mb-12"
        variant="h2"
      >
        Introducing
      </Memo(Typography)>
      <Memo(Typography)
        className="leading-loose max-w-xl"
      >
        Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Cicero's De Finibus Bonorum et Malorum for use in a type specimen book
      </Memo(Typography)>
    </div>
    <div
      className="hidden md:block w-56"
    >
      <Memo(LinkBase)
        className="ml-12 bg-yellow-400 py-6 px-10 hover:opacity-70"
        href="/about"
      >
        Learn more
      </Memo(LinkBase)>
    </div>
  </div>
  <div
    className="w-full px-10 text-center"
  >
    <Image
      alt="introduce"
      height={256}
      layout="responsive"
      src="https://images.prismic.io/nextjsexample/b56dc3ae-2008-424b-9833-5f118c0754c9_intro.jpeg"
      width={976}
    />
  </div>
</div>
`;
